src/components/Favorites/__test__/container.test.js

```diff
@@ -36,9 +36,12 @@ describe('FavoritesContainer ', () => {
   it('Check is calling container method.', () => {
     const wrapper = mount(<FavoritesContainer store={store} />);
     const container = wrapper.find('FavoritesContainer').instance();
-    jest.spyOn(container, 'goToNextPage');
+    jest.spyOn(store, 'dispatch');
```

Comment:
Dispatch is already mocked https://github.com/cyberspacedk/MovieDB/pull/11/files/c08d388142d6dfa45e7bc526be8bd773d8b83b80..a73258b91a5f147f40f20ab7d6dac37a88880ad9#diff-06ce269d62790f4c785a654d343f7bf2R21